app/controllers/api/v1/beers_controller.rb

```diff
@@ -1,10 +1,25 @@
 class Api::V1::BeersController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
+
 
   def index
     render json: Beer.all
   end
 
+  def create
+    newBeerData = JSON.parse(request.body.read)
+
+    newBeer = Beer.new(name: newBeerData["name"], style: newBeerData["style"], description: newBeerData["description"], ABV: newBeerData["ABV"])
+
+    if newBeer.save
+      render json: newBeer
+    else
+      render json: { errors: newBeer.errors.full_messages }, status: :unprocessable_entity
+    end
+  end
+
```

Comment:
You should NOT be using request.body.read. Please use `params`. You will need to review this article if you are having trouble 
https://learn.launchacademy.com/lessons/fetch-in-a-react-application-within-rails